src/build_context.rs

```diff
@@ -118,6 +118,8 @@ pub struct BuildContext {
     pub interpreter: Vec<PythonInterpreter>,
     /// Cargo.toml as resolved by [cargo_metadata]
     pub cargo_metadata: Metadata,
+    /// Whether to use universal2 or use the native macOS tag (off)
+    pub universal2: bool,
```

Comment:
If we defaults to set universal2 to `true`, I think this needs to be changed to an off switch just like `skip_auditwheel`

Reply:
I think it should default to off